src/situations/commun/vues/defi.vue

```diff
@@ -216,7 +216,7 @@ export default {
     },
 
     reponseIntitule () {
-      const reponseId = this.reponse;
+      const reponseId = this.reponse.reponse;
 
       if (this.question.choix) {
         const reponse = this.question.choix.find(choix => choix.id === reponseId);
```

Comment:
Puisque tu as déjà l'objet `reponse`, c'est inutile de le chercher de nouveau

Reply:
L'objet réponse dans `this.reponse` n'est pas celui dans les data.

Exemple je reçois : 
```
this.reponse
{
  reponse: 'jazzADimoudon',
  succes: true,
  score: undefined
} 

reponseData
{
  id: 'jazzADimoudon',
  nom_technique: 'ALrd/jazz_a_dimoudon',
  bonneReponse: true,
  retranscription_audio: "Jazz à Dimoudon"
}

Reply:
Ah ok, mais dans ce cas, est-ce que l'ajout de la retranscription audio ne devrait pas se faire dans un des sous-composants de defis.vue ?